cyclonedx_buildroot/_internal/cli.py

```diff
@@ -174,22 +174,21 @@ def run(*, argv: Optional[Sequence[str]] = None, **kwargs: Any) -> Union[int, No
     br_bom = Bom()
     br_bom.metadata.component = rootComponent = Component(name=args.product_name,
                                                           version=args.product_version)
-    # TODO Specification 1.6 ends support of "manufacture". Remove in future release.
-    br_meta = BomMetaData(manufacture=OrganizationalEntity(name=args.manufacturer_name),
-                          manufacturer=OrganizationalEntity(name=args.manufacturer_name),
+
+    br_meta = BomMetaData(manufacturer=OrganizationalEntity(name=args.manufacturer_name),
```

Comment:
this is definitely a bug! 
the comma(`,`) on the end of the line makes `br_meta` a `tuple` with one element of type `Metadata`.
the intended code was to have `br_meta` being that object of type `Metadata`.

you better get some SCA in place. like https://github.com/CycloneDX/cyclonedx-buildroot/pull/64 and `mypy`


Anyway, get rid of it, and assign the `br_bom.metadata.manufacturer` directly.
the library models, while not perfect, are designed in a way that all collections and data-objects are mandatory - no need to create or overwrite/assign them.
see the docs https://cyclonedx-python-library.readthedocs.io/en/latest/autoapi/cyclonedx/model/index.html